.github/workflows/ci_linux.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             cc: "gcc-10"
```

Comment:
why not just comment-out the coverage block? Seems overkill to add all this logic